packages/elements/src/photon-multirx-form/index.tsx

```diff
@@ -40,414 +42,481 @@ export type Address = {
-          ) {
-            const patient = store.patient?.value;
-            if (patient?.preferredPharmacies && patient?.preferredPharmacies?.length > 0) {
-              // remove the current preferred pharmacy
-              removePatientPreferredPharmacyMutation({
-                variables: {
-                  patientId: patient.id,
-                  pharmacyId: patient?.preferredPharmacies?.[0]?.id
-                },
-                awaitRefetchQueries: false
-              });
-            }
-            // add the new preferred pharmacy to the patient
-            updatePatientMutation({
-              variables: {
-                id: patient.id,
-                preferredPharmacies: [store.pharmacy?.value]
-              },
-              awaitRefetchQueries: false
-            });
-          }
+        dispatchOrderCreated(orderData!.createOrder);
+      }
+    } else {
+      setErrors(errors);
+    }
+  };
 
-          const { data: orderData, errors } = await orderMutation({
-            variables: {
-              patientId: store.patient?.value.id,
-              pharmacyId: props?.pharmacyId || store.pharmacy?.value || '',
-              fulfillmentType: store.fulfillmentType?.value || '',
-              address,
-              fills: prescriptionData?.createPrescriptions.map((x) => ({ prescriptionId: x.id }))
-            },
-            refetchQueries: [],
-            awaitRefetchQueries: false
-          });
+  // decide whether to show the combine modal or submit the form
+  const combineOrSubmit = () => {
+    if (props.enableCombineAndDuplicate && recentOrdersActions.hasRoutingOrder()) {
+      return displayCombineDialog();
+    }
+    return submitForm(props.enableOrder);
+  };
 
-          if (props.enableOrder) {
-            setIsLoading(false);
-          }
-          if (errors) {
-            dispatchOrderError(errors);
-            return;
-          }
-          dispatchOrderCreated(orderData!.createOrder);
-        }
-      } else {
-        setErrors(errors);
-      }
-    };
+  createEffect(() => {
+    if (props.triggerSubmit) {
+      combineOrSubmit();
+    }
+  });
 
-    options.element['submitForm'] = submitForm;
+  createEffect(() => {
+    dispatchPrescriptionsFormValidate(
+      Boolean(
+        props.formStore.draftPrescriptions?.value?.length > 0 && props.formStore.patient?.value
+      )
+    );
+  });
 
-    createEffect(() => {
-      dispatchPrescriptionsFormValidate(
-        Boolean(store.draftPrescriptions?.value?.length > 0 && store.patient?.value)
-      );
-    });
+  let prescriptionRef: HTMLDivElement | undefined;
 
-    createEffect(() => {
-      if (props.triggerSubmit) {
-        submitForm(props.enableOrder);
-      }
-    });
+  return (
+    <div ref={ref}>
+      <style>{tailwind}</style>
+      <style>{shoelaceDarkStyles}</style>
+      <style>{shoelaceLightStyles}</style>
+      <style>{styles}</style>
+      <style>{photonStyles}</style>
```

Comment:
One day someone needs to explain to me:
- Why do we need this all other the place
- Is there a reason they can't all be in one CSS file? or at least one component

Reply:
because of nested shadow doms with web components blocking css. 

I hate it, no more nesting css. There might be a way to fix it short term but the goal is to get to a place where photon-client has no styles and then just the prescribe element has styles at the top of the web component and that's it.

or we just release react components